micro-rdk/src/esp32/entry.rs

```diff
@@ -35,7 +34,7 @@ use crate::{
 // If they are invalid or absent it will start the provisioning server. Once provision is done it invokes the main server.
 async fn serve_async<S>(
     exec: Executor,
-    #[cfg(feature = "provisioning")] info: Option<ProvisioningInfo>,
+    info: ProvisioningInfo,
```

Comment:
Why did the `Option`-ality go away here?

Reply:
reverted back to Option